src/Raven.Server/Documents/ETL/EtlLoader.cs

```diff
@@ -474,9 +474,14 @@ public void HandleDatabaseRecordChange(DatabaseRecord record)
                     }
                 }
             });
+
+            if (_processes.Length > 0)
```

Comment:
@arekpalinski can you review this? LGTM

Reply:
Just to be sure - @karmeli87 this is because we no longer have a distinction between `StateChange` and `ValueChange`, right? 

Although, wouldn't be better off by having dedicated `DatabaseRecordChange.EtlStateChange` and then separate flow for that. Please check my comment below. If you agree that introducing dedicated change for that it's good idea then we should have the same for Subsctiptions - a subscription state is handled the same way

Reply:
If this change is (logically) correct and doesn't worsen performance, I think we should do a finer granularity later.

Reply:
Current implementation is correct. Introducing finer granularity is an enhancement that can be implemented later.